README.md.template

```diff
@@ -39,6 +39,11 @@ public class BarberView extends FrameLayout {
     @StyledAttr(value = R.styleable.BarberView_stripeColor, kind = Kind.COLOR)
     public int stripeColor;
 
+    // alternately if you use support annotations, the kind can be inferred
+    // @StyledAttr(R.styleable.BarberView_stripeColor)
```

Comment:
Hmm, thinking about this, let's actually just remove the `COLOR` kind all together. If people don't use support annotations at this point, well, their problem :)